catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/CachedResourceMetacardComparator.java

```diff
@@ -52,35 +49,18 @@
   private static final List<Function<Metacard, ?>> METACARD_METHODS =
       ImmutableList.of(
           Metacard::getModifiedDate,
-          Metacard::getThumbnail,
-          Metacard::getResourceURI,
-          Metacard::getLocation,
-          Metacard::getEffectiveDate,
           Metacard::getCreatedDate,
-          Metacard::getExpirationDate,
-          Metacard::getContentTypeName);
-
-  private static final Set<String> ATTRIBUTES_TO_IGNORE =
-      ImmutableSet.of(
-          Metacard.CHECKSUM,
-          Metacard.MODIFIED,
-          Metacard.RESOURCE_SIZE,
-          Metacard.THUMBNAIL,
-          Metacard.RESOURCE_URI,
-          Metacard.GEOGRAPHY,
-          Metacard.EFFECTIVE,
-          Metacard.CREATED,
-          Metacard.EXPIRATION,
-          Metacard.CONTENT_TYPE);
+          (m) -> m.getAttribute(Core.METACARD_MODIFIED),
+          (m) -> m.getAttribute(Core.METACARD_CREATED));
 
   private CachedResourceMetacardComparator() {}
 
   /**
    * Indicates whether the resource associated with the provided metacard has changed by comparing
    * the state of the metacard when it was cached with the state of the metacard when it was last
-   * retrieved. Since there is no single attribute that clearly indicates whether the resource has
-   * changed or not, this class compares as many attributes as possible and if it find a difference,
-   * it will return {@code false}.
+   * retrieved. The main attributes to be compared to indicate whether a resource has changed or not
+   * are checksum, modified date, and created date. If this finds a difference in these values it
```

Comment:
✏️ Should probably mention that we compare both, the metacard _and_ resource creation and modified dates.

Reply:
fixed